README.md

```diff
@@ -95,6 +95,7 @@ This repository contains algorithms and data structures implemented in C# for ed
 		* [Luhn](./Algorithms/Other/Luhn.cs)
 		* [Mandelbrot](./Algorithms/Other/Mandelbrot.cs)
 		* [Koch Snowflake](./Algorithms/Other/KochSnowflake.cs)
+		* [RGB-HSV Conversion](./Algorithms/Other/RGBHSVConversion.cs)
```

Comment:
Could you also update https://github.com/TheAlgorithms/C-Sharp/blob/master/DIRECTORY.md ? It's needed for our website to work properly.